src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -139,7 +153,7 @@ function ReportActionItemMessageEdit(props) {
             // to prevent the main composer stays hidden until we swtich to another chat.
             ComposerActions.setShouldShowComposeInput(true);
         };
-    }, [props.action.reportActionID]);
+    }, [props.action.reportActionID, isMobileSafari]);
```

Comment:
`isMobileSafari` const won't change during the lifecycle of the component. Do we need this to make lint happy?

Reply:
Yes lint 😢